### ISSUE TITLE
deal with the JVM logging craziness

### ISSUE THREAD

**Issue by toth-istvan-zoltan** [spxbhuhb/adaptive#4]
JVM logger related problems:

* two lines per entry
* not the same logger as Ktor and everything else
* wrong name (should be set by the compiler plugin, but I think I don't process workers and that is the missing part)

```text
May 14, 2024 4:50:26 AM hu.simplexion.adaptive.log.JvmLogger info
INFO: no change in the last minute

2024-05-14 04:50:21.742 [DefaultDispatcher-worker-3] INFO  ktor.application - Responding at http://0.0.0.0:8080
```

**Comment by toth-istvan-zoltan** [comment]
Also see: https://youtrack.jetbrains.com/issue/KTOR-5227

As for now, logback is on 1.2.11 to avoid runtime troubles on Android. I should investigate this a bit deeper to find a good solution (or just use 1.2.11 for a while).

**Comment by toth-istvan-zoltan** [comment]
The original problems are solved, logback 1.2.11 is still there.

**Comment by toth-istvan-zoltan** [comment]
Updated logback to 1.5.6, seems to be working on Android as well.
